app/views/carts/index.html.haml

```diff
@@ -0,0 +1,33 @@
+%body.cart-body
+  .title-logo
+    = link_to home_path do
+      %span.first-word
+        Canteen
+      %span.second-word
+        Koala
+  %h2.cart-title
+    Welcome to your cart
+  .cart-list
+    - @cart.each do |index, product|
+      .cart-item
+        = cl_image_tag product['image_key']
```

Comment:
would be cool to move this to db models :-)

Reply:
Sure thing, will do this transition for the upcoming assignments👌